### PR TITLE
Accordion gap

### PR DESCRIPTION
**Pull request by MalcolmVonMoJ**
Added new if statement to Accordion block to only display first paragraph if it is actually containing anything.

### CODE REVIEW

File: src/custom-blocks/accordion/index.php [hunk 1]
```diff
@@ -88,10 +88,13 @@ function render_callback_accordion_block_section($attributes, $content)
...
         <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
-            <p class="govuk-body">
-                <?php _e(esc_html($attribute_accordion_section_TextArea)) ; ?>
-                <?php _e(esc_html($content)); ?>
-            </p>
+            <?php
+            $firstParagraphContent = _(trim(esc_html($attribute_accordion_section_TextArea)));
+            if (!empty($firstParagraphContent)) {
+                echo "<p class='govuk-body'>$firstParagraphContent</p>";
+            }
```
Comment (RobLoweMOJ) [inline src/custom-blocks/accordion/index.php:95]: Should  _e(esc_html($content));  be in a p tag?

Comment (MalcolmVonMoJ) [reply src/custom-blocks/accordion/index.php:95]: No, as it could be a list or other block (I think just a list actually, but it might not be a p)

**Comment by MalcolmVonMoJ** [comment]
### A note on wrapping everything in a `<p>` tag:
Previously the code was rendering thus:
```
<div id="hale-accordion-content-4" class="govuk-accordion__section-content" aria-labelledby="hale-accordion-heading-4">
     <p class="govuk-body">
        Single line in section content area                
    </p>
    <p>And another paragraph added as a block</p>
    <p></p>
</div>
```
The empty `<p>` was caused by wrapping everything in a single `<p>` tag.  The blocks were adding their own paragraph markers therefore the browser was creating more.  the first `</p>` and the last `<p>` were inserted by the browser.  Viewing the page source demonstrates that they are not there:
```
  <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
    <p class="govuk-body">
    Single line in section content area
   
<p>And another paragraph added as a block</p>
   
    </p>
  </div>
```

This is fixed by only wrapping the section content area in `p` tags and moving the block content out of it.  
The new code renders thus:
```
<div id="hale-accordion-content-4" class="govuk-accordion__section-content" aria-labelledby="hale-accordion-heading-4">
    <p class="govuk-body">Single line in section content area</p>
    <p>And another paragraph added as a block</p>
</div>
```
Which is in line with the source:
```
  <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">

    <p class='govuk-body'>Single line in section content area</p>

<p>And another paragraph added as a block</p>

  </div>
```

**Review by RobLoweMOJ — APPROVED** [review]
LGTM

**Pull request merged**